### PR TITLE
Simplify Chinese translation of Scala Tour: operators.md

### PR DESCRIPTION
**Pull request by realwunan**
Simplify Chinese translation of Scala Tour: Operators

### CODE REVIEW

File: _zh-cn/tour/operators.md [hunk 1]
```diff
@@ -13,3 +13,69 @@ language: zh-cn
 next-page: by-name-parameters
 previous-page: type-inference
 ---
+在Scala中，运算符即是方法。 任何具有单个参数的方法都可以用作 _中缀运算符_。 例如，可以使用点号调用 `+`：
+```
+10.+(1)
+```
+
+但是，用作中缀运算符更容易理解:
```
Comment (dongxuwang) [inline _zh-cn/tour/operators.md]: 而中缀运算符则更易读：

Comment (realwunan) [reply _zh-cn/tour/operators.md]: Yes, thanks a lot.

**Comment by SethTisue** [comment]
I think a rebase will fix the CI failure

**Comment by realwunan** [comment]
Thanks, Seth Tisue, the build fixed after rebase

**Comment by SethTisue** [comment]
thank you for the contribution!

**Pull request merged**